WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -324,16 +336,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
 
         // Enable the block editor on sites created on mobile
-        switch (requestCode) {
-            case RequestCodes.CREATE_SITE:
-                if (data != null) {
-                    int newSiteLocalID = data.getIntExtra(
-                            SitePickerActivity.KEY_SITE_LOCAL_ID,
-                            SelectedSiteRepository.UNAVAILABLE
-                    );
-                    SiteUtils.enableBlockEditorOnSiteCreation(mDispatcher, mSiteStore, newSiteLocalID);
-                }
-                break;
+        if (requestCode == RequestCodes.CREATE_SITE) {
+            if (data != null) {
```

Comment:
💡 Can we combine these 2 `if` statements? It doesn't look like we do anything different if `requestCode` is `CREATE_SITE` and `data` is `null`, so maybe this works:

```suggestion
        if (requestCode == RequestCodes.CREATE_SITE && data != null) {
```